### PR TITLE
Added Network detail page

### LINKED ISSUE #774: `network detail screen`
#### What would you like to be added:
From header, when you click, it will open this screen for more details. Last sync will be coming from interval polling.

#### Why is this needed:
https://www.figma.com/proto/SDIpZciuxJlUf4xf8yheiY/Light-Wallet?page-id=287%3A11410&node-id=659%3A12498&viewport=241%2C48%2C0.8&scaling=scale-down&starting-point-node-id=665%3A12670&show-proto-sidebar=1
Screenshot 2021-09-02 at 5 08 16 PM

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:
From header, when you click, it will open this screen for more details. Last sync will be coming from interval polling.

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #774

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

File: app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+              style={tailwind(`h-2 w-2 rounded-full ${connected ? 'bg-green-500' : 'bg-red-500'} mr-1.5`)}
+              testID='status_icon_network'
+            />)
+        }
+      ]
+    }, {
+      key: NETWORK.NetworkDetails,
+      data: [
+        {
+          label: translate('screens/NetworkDetails', 'Last Sync'),
```
Comment (thedoublejay) [inline app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx:51]:
```suggestion
          label: translate('screens/NetworkDetails', 'Last synced'),
```

File: app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+          label: translate('screens/NetworkDetails', 'Total Masternodes'),
+          status: masternodeCount,
+          testID: 'total_masternodes'
+        }
+      ]
+    }
+  ]
+
+  return (
+    <ThemedScrollView testID='network_details'>
```
Comment (thedoublejay) [inline app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx:68]: You can remove sectionlist here and just use scrollview

File: app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+export function NetworkDetails (): JSX.Element {
+  const { network } = useNetworkContext()
+  const { connected, count: blockCount, masternodeCount, lastSync } = useSelector((state: RootState) => state.block)
+  const syncFormattedDate = (lastSync != null) ? dayjs(lastSync).format('MMM D, h:mm a') : ''
+
+  const statsData = [
+    {
+      key: NETWORK.ConnectedStatus,
```
Comment (thedoublejay) [inline app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx:30]: ConnectedStatus doesn't look aligned on mobile

Comment (thedoublejay) [reply app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx:30]: image

The green dot should be aligned in the middle of Connected

Comment (thedoublejay) [reply app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx:30]: and maybe increase the size a bit

File: app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+          }
+        }}
+        sections={statsData}
+        testID='network_detail_screen_list'
+      />
+    </ThemedScrollView>
+  )
+}
+
+function StatItemRow ({ label, status, testID, statusIcon }: StatItemProps): JSX.Element {
```
Comment (thedoublejay) [inline app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx:101]: Use TextRow or NumberRow here. There's an existing component for it

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Comment by thedoublejay** [comment]
@fullstackninja864 next steps would be to add e2e test cases for it. Some cases to handle:

- [ ] Change network, see if value updates
- [ ] Check if values are same from API side

File: app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx (hunk 3 above)
Comment (fullstackninja864) [reply app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx]: Will make the changes

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**